### PR TITLE
Basic CSS Added

### PR DESCRIPTION
**Pull request by XTech2K**

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+
+        <title>COS IRC Home</title>
+
+        <style>
+            .link {
+                font-size: 1.25em;
+                margin-bottom: 20px;
+            }
+        </style>
+    </head>
```
Comment (asmacdo) [inline templates/home.html]: @XTech2K can you switch the class name to something more specific? "link" is something that is likely to interfere with the global namespace if we expand the page.

File: templates/register.html [hunk 2]
```diff
@@ -15,11 +32,11 @@
...
-        <div style="margin-bottom:18px;">Username/Nick:*  <input type="text" name="username" id="username" value="">    (only letters, numbers, dashes and underscores; between 6 and 16 characters)</div>
-        <div style="margin-bottom:18px;">Password:**      <input type="password" name="pwd1" id="pwd1" value="">    (only letters and numbers; between 6 and 16 characters)</div>
-        <div style="margin-bottom:18px;">Verify Password: <input type="password" name="pwd2"  id="pwd2" value=""></div>
+        <div>Username/Nick:*  <input class="field" type="text" name="username" id="username" value=""></div><div class="note">(only letters, numbers, dashes and underscores; between 6 and 16 characters)</div>
+        <div>Password:**      <input class="field" type="password" name="pwd1" id="pwd1" value=""></div><div class="note">(only letters and numbers; between 6 and 16 characters)</div>
+        <div>Verify Password: <input class="field" type="password" name="pwd2"  id="pwd2" value=""></div>
 
-        <div><input type="submit" name="submit" value="Submit" id="submit"></div>
+        <div style="margin-top: 14px"><input type="submit" name="submit" value="Submit" id="submit"></div>
```
Comment (asmacdo) [inline templates/register.html]: @XTech2K same as above. Use a more specific name for css classes.

**Comment by kmjungersen** [comment]
@XTech2K:  first, good work!!  Something we need to keep in mind though is that all styling like this should be moved to it's own CSS file in a 'static' directory.  This is pretty standard practice for sites, and I don't think this should be any different.  The rationale for doing so is that it makes it a lot easier to standardize the styling of pages.  We will want this to look and feel like the rest of the OSF in the near future, even just for internal use, and it also makes it easier for someone else who is using this registration server to customize their styling.  Again though, good work!

Also on a related note, our JavaScript should also be moved to a separate static file for additional ease of reading.  This is something @asmacdo and I had discussed and weren't sure about, but the more complicated this gets the more I see the need to separate these aspects.

File: templates/home.html (hunk 1 above)
Comment (kmjungersen) [inline templates/home.html]: Agreed.  This goes along with my overall comments below.

**Comment by kmjungersen** [comment]
One more important note: 'home.html' should be renamed 'index.html'.  This is picky, but the default page of a site is almost always 'index.html', and other developers will expect this and be looking for it.  'Register.html' is fine, but the default navigation page should still be 'index'.

**Pull request merged**